op-chain-ops/cmd/check-delta/aggregate-channels.sh

```diff
@@ -59,12 +60,13 @@ for file in "$directory_path"/*.json; do
         ((ready_channel_count++))
     fi
     # check channels contain invalid batches
-    if [ $(jq -r ".invalid_batches" "$file") == "true" ] ; then
+    if [ "$(jq -r ".invalid_batches" "$file")" == "true" ] ; then
         channels_with_invalid_batches+=("$file")
     fi
     # count singular batch count and span batch count
     jq_result=$(jq "$batch_type_counter_jq_script" "$file")
-    read span_batch_count_per_channel singular_batch_count_per_channel <<< $jq_result
+    # shellcheck disable=SC2162
+    read span_batch_count_per_channel singular_batch_count_per_channel <<< "$jq_result"
```

Comment:
[SC2162](https://www.shellcheck.net/wiki/SC2162) is "read without -r will mangle backslashes". I'm not sure what content is being read here, so to avoid breaking changes I disabled this check, but perhaps adding `-r` is desirable?

Reply:
I'm not sure what `-r` does, but it may be fine.

Reply:
Without `-r` backslashes are treated as escape characters, and with `-r` they are read directly and not treated as escape characters.

AFAICT this jq result is just two integers counted using the `batch_type_counter_jq_script`, so disabling this should be safe